libtextsecure/outgoing_message.js

```diff
@@ -460,20 +491,74 @@ OutgoingMessage.prototype = {
     };
   },
   // Send a message to a public group
-  sendPublicMessage(number) {
-    return this.transmitMessage(
-      number,
-      this.message.dataMessage,
-      this.timestamp,
-      0 // ttl
-    )
-      .then(() => {
-        this.successfulNumbers[this.successfulNumbers.length] = number;
-        this.numberCompleted();
-      })
-      .catch(error => {
-        throw error;
-      });
+  async sendPublicMessage(number) {
+    if (this.isPublic) {
```

Comment:
how about if (!this.isPublic) throw warning and return?

Reply:
I think I missed this when I merged clearnet. I don't think `if (this.isPublic)` is necessary, I will just remove it.